.github/workflows/codeql.yml

```diff
@@ -0,0 +1,92 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "upcoming-release/*" ]
+  pull_request:
+    branches: [ "master", "upcoming-release/*" ]
+  schedule:
+    - cron: '40 13 * * 4'
```

Comment:
```suggestion
    - cron: '0 0 * * 1-5'
```
Run every weekday?

Reply:
Well I think we may commit code on weekends so perhaps not.